spec/helpers.rb

```diff
@@ -222,7 +222,7 @@ def create_and_review_incident
 
   def submit_to_state
     visit_status :state_submission
-    click_button 'FINAL SUBMISSION'
+    click_button 'SUBMISSION'
```

Comment:
The problem is that, depending on branding, the button may say "FINAL SUBMISSION" or "FINAL STATE SUBMISSION".